corehq/apps/app_manager/models.py

```diff
@@ -2895,7 +2895,8 @@ def get_phase(self):
                     None)
 
     def disable_schedule(self):
-        self.schedule.enabled = False
+        if self.schedule:
+            self.schedule.enabled = False
         phase = self.get_phase()
```

Comment:
Curious why this was necessary... I think all advanced forms always have the `schedule` property, and I think this is only ever called on advanced forms?

Reply:
`schedule` is there, but it defaults to `None`